### ISSUE TITLE
Accessibility docs: disabled state

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#2261]
**TLDR;** don't use disabled states

Disabled states of input fields or buttons pose many accessibility issues. 
- Disabled fields cannot be reached by pressing tab
- It's often unclear why something is disabled
- It's often unclear how to enable a disabled field or button
- Disabled fields have low contrast
- The disabled state of a field is not always obvious (hard to discern from an enabled field)

### Disabled button on forms with required fields

**Issue:**
Forms with required fields tend to have the send button disabled until all required fields are correctly filled. The issue here is that with long forms or on small devices it's sometimes hard to spot the one field that was missed.

**Solution:**
Let the user make mistakes and help them to correct them. Mark the required fields with error messages, scroll to the first invalid field and help the user fill the form correctly.

### Disabled button on form submit

**Issue:**
The user submits a form. While the client waits for the server response and to prevent double submission, the submit button gets disabled. The reason why this button gets disabled might not be obvious.

**Solution:**
Provide a loading indicator or display a message that shows current progress when uploading large files. Disable duplicate form submissions internally. Inform users that the form was already submitted if they continue to click submit and offer a remedy if the request timed out or an error occured on the server (reload the page, retry form submission, ...).

### Disabled select options

**Issue:**
Some options in a radio button group or select field are disabled because they are not available at this time or with the current configuration. This can be confusing because it's often unclear why these options are not available or how to enable them.

**Solution:**
There are multiple possible solutions, depending on the situation:
- Don't display options that cannot be selected and reduce clutter in your UI
- If you need to inform the user that there are potentially more possible options, show instructions on how to enable these options
- Display read-only data as text and not in disabled form fields

### Disabled tabs
**Issue:**
There are views in a tab element that are currently unavailable or show no data.

_To be continued_

```[tasklist]
### Tasks
- [x] Write accessibility docs for the disabled state
- [x] If possible, create templates or showcases that offer code samples on how to implement the solutions, link them in the docs
- [ ] Review with UX
```
